src/ring_jwt_middleware/core.clj

```diff
@@ -128,7 +128,7 @@
                                          default-jwt-lifetime-in-sec)
                                      long-lived-jwt?)]
          checks (if (fn? jwt-check-fn)
-                  (or (try (jwt-check-fn jwt)
+                  (or (try (seq (jwt-check-fn jwt))
```

Comment:
How you know/ensure that `jwt-check-fn` will return a collection?
For instance, `(seq true)` will throw an exception